src/data/services/EnterpriseDataApiService.js

```diff
@@ -5,35 +5,39 @@ import { getAccessToken } from '../../utils';
 
 class EnterpriseDataApiService {
   // TODO: This should access the data-api through the gateway instead of direct
-  static enterpiseBaseUrl = `${config.DATA_API_BASE_URL}/enterprise/api/v0/enterprise/`;
+  static enterpriseBaseUrl = `${config.DATA_API_BASE_URL}/enterprise/api/v0/enterprise/`;
 
   static fetchCourseEnrollments(enterpriseId, options) {
     const queryParams = {
       page: 1,
       page_size: 50,
       ...options,
     };
-    let enrollmentsUrl = `${this.enterpiseBaseUrl}${enterpriseId}/enrollments/`;
+    const enrollmentsUrl = `${this.enterpriseBaseUrl}${enterpriseId}/enrollments?${qs.stringify(queryParams)}`;
     const jwtToken = getAccessToken();
 
-    if (options && typeof options === 'object') {
-      enrollmentsUrl += `?${qs.stringify(queryParams)}`;
-    }
-
     return axios.get(enrollmentsUrl, {
-      withCredentials: true,
```

Comment:
Because we're using token authentication in the header we don't need to pass along any cookies, so withCredentials wasn't needed and would have required extra CORS configurations on the data-api if we kept it.